axlearn/common/trainer.py

```diff
@@ -909,6 +910,7 @@ def restore_checkpoint(self, restore_step: Optional[int] = None) -> Optional[int
             restore_input_iter = cfg.save_input_iterator
             try:
                 # Try to restore with `input_iter`.
+                self._maybe_record_event(measurement.Event.START_DATA_LOADING)
```

Comment:
Does "DATA_LOADING" include restoring checkpoints? It looks a little bit confusing to me

Reply:
Data loading should include just that; the loading of each batch of data pre-training or during/overlapped with training.

Reply:
right, so should we remove it from "restore_checkpoint" here?